contributed_definitions/nyaml/NXcircuit_board.yaml

```diff
@@ -0,0 +1,17 @@
+category: base
+doc: |
+  Circuit board with e.g. ADC and/or DAC electronic components.
+  
+  Currently used to store the settings of the so-called magboards used in
+  Nion electron microscopes but likely this could be a useful base class for
+  substantially more use cases where details at a deep technical instrument design
+  level are relevant or important.
+symbols:
+  doc: The symbols used in the schema to specify e.g. dimensions of arrays.
+NXcircuit_board:
```

Comment:
if it is a specific board, maybe its name can also reflect that. In the future we may want to base it on a more generic NXcircuit from the spm branch